.github/CODEOWNERS

```diff
@@ -57,9 +57,10 @@
 /fiddler/                                @dannybrock sales@fiddler.ai @DataDog/ecosystems-review
 /filebeat/                               @wk8 jean@tripping.com
 /filemage/                               @devopsec
-/finout/                                 @roiravhon support@finout.io @DataDog/ecosystems-review
```

Comment:
Can you rebase with master? `marketplace-review` recently changed to `ecosystems-review` so the CODEOWNERS file is out of sync.

Reply:
OK that's done